default_warehouse_from_sale_team/i18n/es.po

```diff
@@ -26,51 +31,109 @@ msgid "Access to all pickings"
+msgid "Default Team"
+msgstr "Equipo por defecto"
+
+#. module: default_warehouse_from_sale_team
+#: model:ir.model,name:default_warehouse_from_sale_team.model_default_warehouse
+#: model:ir.model.fields,field_description:default_warehouse_from_sale_team.field_crm_team__default_warehouse
 #: model:ir.module.category,name:default_warehouse_from_sale_team.default_warehouse_module
 msgid "Default Warehouse"
 msgstr "Almacén por defecto"
 
 #. module: default_warehouse_from_sale_team
-#: field:default.warehouse,display_name:0
+#: model:ir.model.fields,field_description:default_warehouse_from_sale_team.field_default_picking_type__display_name
+#: model:ir.model.fields,field_description:default_warehouse_from_sale_team.field_default_warehouse__display_name
 msgid "Display Name"
 msgstr "Nombre para mostrar"
 
 #. module: default_warehouse_from_sale_team
-#: field:default.warehouse,id:0
+#: model:ir.model.fields,field_description:default_warehouse_from_sale_team.field_default_picking_type__id
+#: model:ir.model.fields,field_description:default_warehouse_from_sale_team.field_default_warehouse__id
 msgid "ID"
 msgstr "ID"
 
 #. module: default_warehouse_from_sale_team
-#: help:crm.case.section,default_warehouse:0
+#: model:ir.model.fields,help:default_warehouse_from_sale_team.field_crm_team__default_warehouse
 msgid "In this field can be defined a default warehouse for the related users to the sales team."
 msgstr "En este campo se puede definir el almacén por defecto para los usuarios relacionados al equipo de ventas."
 
 #. module: default_warehouse_from_sale_team
-#: field:default.warehouse,__last_update:0
+#: model:ir.model,name:default_warehouse_from_sale_team.model_account_invoice
+msgid "Invoice"
+msgstr "Factura"
+
+#. module: default_warehouse_from_sale_team
+#: model:ir.model.fields,help:default_warehouse_from_sale_team.field_crm_team__journal_stock_id
+msgid "It indicates journal to be used when move line is created withthe warehouse of this sale team"
+msgstr "Indica el diario que se utilizará cuando se cree la línea de movimiento con el almacén de este equipo de ventas"
+
+#. module: default_warehouse_from_sale_team
+#: model:ir.model,name:default_warehouse_from_sale_team.model_account_journal
+msgid "Journal"
+msgstr "Diario"
+
+#. module: default_warehouse_from_sale_team
+#: model:ir.model.fields,field_description:default_warehouse_from_sale_team.field_crm_team__journal_stock_id
+msgid "Journal stock valuation"
+msgstr "Diario para valoración de existencias"
+
+#. module: default_warehouse_from_sale_team
+#: model_terms:ir.ui.view,arch_db:default_warehouse_from_sale_team.view_crm_team_form_inherit
+msgid "Journal's sale team"
+msgstr "Diarios de equipo de ventas"
+
+#. module: default_warehouse_from_sale_team
+#: model:ir.model.fields,field_description:default_warehouse_from_sale_team.field_crm_team__journal_team_ids
+msgid "Journal's sales teams"
+msgstr "Diarios de equipos de ventas"
```

Comment:
This should be with the above term, whose only difference is sales team vs sale team

Reply:
Done